From 2835a2232631f91b4b6c78221a9c392471eef387 Mon Sep 17 00:00:00 2001
From: Stuart MENEFY <stuart.menefy@st.com>
Date: Wed, 13 May 2009 12:37:52 +0100
Subject: [PATCH] sh_stm: Fix cache writethrough option name

The config option used to select write through caches on SH changed some
time ago, so update this change (introduced by an ST specific patch)
to match.

Signed-off-by: Stuart Menefy <stuart.menefy@st.com>
---
 arch/sh/mm/cache-sh4.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/sh/mm/cache-sh4.c b/arch/sh/mm/cache-sh4.c
index 2c92bf5..cf9fae6 100644
--- a/arch/sh/mm/cache-sh4.c
+++ b/arch/sh/mm/cache-sh4.c
@@ -663,7 +663,7 @@ static void __flush_dcache_segment_1way(unsigned long start,
 	} while (a0 < a0e);
 }
 
-#ifdef CONFIG_SH_WRITETHROUGH
+#ifdef CONFIG_CACHE_WRITETHROUGH
 /* This method of cache flushing avoids the problems discussed
  * in the comment above if writethrough caches are enabled. */
 static void __flush_dcache_segment_2way(unsigned long start,
-- 
1.6.0.6

